guide/payments/request.md

```diff
@@ -116,82 +88,128 @@ A piece of meta data unique to invoices is an expiration time. This is how long
+More on the [address types]({{ "/guide/glossary/address/" | relative_url }}) and [interoperability]({{ "/guide/designing-products/wallet-interoperability/" | relative_url }}) page.
 
-Several, not widely supported, reusable payments request formats also exist. For Lightning, there are BOLT 12 offers, AMP invoices and Keysends, which require just a node ID. For on-chain, there are reusable payment codes [BIP 47](https://bips.xyz/47) and [xpubs]({{ "/guide/glossary/#extended-public-key-xpub-ypub-zpub" | relative_url }}), which can be used to generate addresses for receiving bitcoin on-chain. Learn more about these on our [payment request formats]({{ "/guide/payments/request/payment-request-formats/" | relative_url }}) page.
+</div>
 
-### Contact cards
+### Meta data
 
-Contact cards are a great way to  store, manage and share reusable payment requests. A user's own contact card can save and share the users own reusable payment requests. Contact cards of other users can store their reusable payment requests shared with the user. This greatly simplifies both requesting and [sending]({{ "/guide/payments/send/" | relative_url }}) flows.
+A payment request with only standard data, like an amount and date, communicates little to both parties about purpose and context of the payment.
 
-<div class="image-slide-gallery">
+<div class="center" markdown="1">
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/Home.png"
-   retina = "/assets/images/guide/payments/request/Home@2x.png"
-   layout = "shadow"
-   caption = "Users should be able to access their profile via the home page."
-   alt-text = "Screen showing home screen number pad entry."
+   image = "/assets/images/guide/payments/request/single-use-payment-request-settings.png"
+   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings@2x.png"
+   layout = "float-left-desktop -background -shadow"
+   caption = "Let users attach payment meta data to single-use payment requests."
+   alt-text = "Screen showing single payment request settings."
    width = 250
    height = 541
 %}
 
+Users should have the option to attach a note for record-keeping and for the sender to read, tags, labels and any other metadata that may be relevant to give more context to the payment.
+
+This meta data should also be able to be [backed up]({{ "/guide/onboarding/backing-up-a-wallet/" | relative_url }}) in case the user loses access to their wallet and has to restore it.
+
+</div>
+
+## Reusable payment requests
+
+Reusable payment requests offer an improved user experience, more flexibility, and enable unique use cases compared to single-use payment requests. Reusable payment requests avoid users having to manually generate a payment request each time they need to receive bitcoin.
```

Comment:
```suggestion
Reusable payment requests offer an improved user experience, more flexibility, and enable unique use cases, as they don't need to be re-generated every time the user wants to receive bitcoin.
```
Simplifying to avoid repetition of the term `payment request`.